R/PipeOpTaskSurvClassif.R

```diff
@@ -0,0 +1,151 @@
+#' @title PipeOpTaskSurvClassif
```

Comment:
Food for thought: since we will be adding more of these pipeops that convert surv tasks to classif, maybe consider renaming this to eg `PipeOpTaskSurvClassifDiscTime`? I am thinking that adding a `method` parameter might not be the best choice as these methods might output different channels...